README.md

```diff
@@ -30,22 +30,6 @@ Also note that as of version `1.2.0` you no longer have to provide the required
 
 The following environment variables may be used in place of parameters: `SENTRY_API_KEY`, `SENTRY_AUTH_TOKEN`, `SENTRY_ORG_SLUG`, and `SENTRY_PROJECT_SLUG`.
 
-#### Uploading Symbolication Files
```

Comment:
I agree that we should avoid this duplication given `upload-dif` is the replacement. I'm not sure though if we should keep the lane and deprecate it, so we don't break old configs that did not migrate yet. What do you think @philipphofmann?

Reply:
So @bruno-garcia and @philipphofmann you also agree that we can remove this? Users will have to update their `Fastlane` files if we do so, so we'd need to make clear that this is a breaking change in the next release.

Reply:
Would it be possible to have `sentry_upload_dsym` simply forward the call to `sentry_upload_dif` to avoid the breaking change? Not sure it's worth creating friction with customers that have set this up 5 years ago and have no problems